src/main/java/uk/gov/hmcts/divorce/common/config/advice/GlobalExceptionHandler.java

```diff
@@ -52,4 +53,11 @@
             String.format("%s - %s", exception.getMessage(), exception.contentUTF8())
         );
     }
+
+    @ExceptionHandler(DocumentDownloadException.class)
+    public ResponseEntity<Object> handleBadRequestException() {
```

Comment:
method name seems incorrect

Reply:
Any suggestions?

Reply:
Changed it to - handleDocumentDownloadException